content/rc/how-to/flush-db.md

```diff
@@ -35,4 +35,41 @@ includes all data in memory and persisted to disk. So before doing
 flushall, it is best practice to [backup your
 database]({{< relref "/rc/administration/configure/backups.md" >}}) first.
 
-### 
+### Flushing Redis Enterprise Cloud CRDBs
+
+When you flush a CRDB, multiple resources flush their data simultaneously.
+To flush data from a CRDB, use the flush command in the crdb-cli.
+
+1. To find the ID of the CRDB, run:
+
+    ```src
+    crdb-cli crdb list
+    ```
+
+1. To flush the CRDB, run:
+
+    ```src
+    crdb-cli crdb flush --crdb-guid <ID_of_the_CRDB>
+    ```
+
```

Comment:
Don't we want here to have a command line for tracking the task status or the crdb-cli is not mature enough in this respect?

Reply:
Added